src/content/en/fundamentals/accessibility/focus/using-tabindex.md

```diff
@@ -0,0 +1,417 @@
+the natural tab order. If there are multiple elements with a tabindex greater
+than 0, the tab order starts from the lowest value that is greater than zero and
+works its way up. Using a tabindex greater than 0 is considered an
+**anti-pattern**.
+
+    <button>I should be first</button>
+    <button>And I should be second</button>
+    <button tabindex="5">But I jumped to the front!</button>
+
+{% framebox height="80px" %}
+<button>I should be first</button>
+<button>And I should be second</button>
+<button tabindex="5">But I jumped to the front!</button>
+{% endframebox %}
+
+This is particularly true of non-input elements like headers, images, or article
+titles. Adding `tabindex` to those kinds of elements is counter-productive. If
+possible, it's best to arrange your source code so the DOM sequence provides a
+logical tab order. If you do use `tabindex`, restrict it to custom interactive
+controls like buttons, tabs, dropdowns, and text fields; that is, elements the
+user might expect to provide input to.
+
+Don't worry about screen reader users missing important content because it
+doesn't have a `tabindex`. Even if the content is very important, like an image,
+if it's not something the user can interact with, there's no reason to make it
+focusable. Screen reader users can still understand the content of the image so
+long as you provide proper `alt` attribute support, which we'll cover shortly.
+
+## Managing focus at the page level
+
+Here's a scenario where `tabindex` is not only useful, but necessary. You might
+be building a robust single page with different content sections, not all of
+which are simultaneously visible. In this kind of page, clicking a navigation
+link might change the visible content without doing a page refresh.
+
+When this happens, you would probably identify the selected content area, give
+it a `tabindex` of -1 so that it doesn't appear in the natural tab order, and
+call its `focus` method. This technique, called *managing focus*, keeps the
+user's perceived context in sync with the site's visual content.
+
+## Managing focus in components
+
+Managing focus when you change something on the page is important, but sometimes
+you need to manage focus at the control level &mdash; for example, if you're
+building a custom component.
+
+Consider the native `select` element. It can receive basic focus but, once
+there, you can use the arrow keys to expose additional functionality (the
+selectable options). If you were building a custom `select` element, you would
+want to expose these same kinds of behaviors so that users who rely primarily on
+the keyboard could still interact with your control.
+
+    <!-- Focus the element using Tab and use the up/down arrow keys to navigate -->
+    <select>
+      <option>Aisle seat</option>
+      <option>Window seat</option>
+      <option>No preference</option>
+    </select>
+
+{% framebox height="80px" %}
```

Comment:
For simple html like this, you don't need to wrap it in a frame box. You only need to wrap in a framebox if there's JavaScript